srtcore/api.cpp

```diff
@@ -771,11 +771,11 @@ int CUDTUnited::newConnection(const SRTSOCKET listen, const sockaddr_any& peer,
 #if ENABLE_LOGGING
        static const char* why [] = {
            "UNKNOWN ERROR",
-           "CONNECTION REJECTED",
+           "INTERNAL REJECTION",
```

Comment:
Changed because the effective `connection rejected due to: CONNECTION REJECTED` sounds a little bit stupid :)

Reply:
Rejected means rejected :)

Reply:
1 stands for `SRT_REJ_SYSTEM`
2 is `SRT_REJ_PEER` (line 641)

Reply:
These numbers have nothing to do with the index here. See how the `why` value is defined in this function.

Reply:
614, 641 and 733 lines update the value of `error`. Not to be confused by `w_error`, which is a parameter of this function and has a completely different meaning.